pkgs/development/python-modules/ihatemoney/default.nix

```diff
@@ -93,7 +93,7 @@ buildPythonPackage rec {
     )
     flask-restful
     flask-talisman
-    flask_wtf
+    flask_wtf.wtforms2
```

Comment:
I think we should check if we can fetch https://github.com/spiral-project/ihatemoney/pull/916.